app/views/users/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Users show method</h1>
\ No newline at end of file
+<h1>Users show method</h1>
+
+  <%= render partial: "shared/user_card" %>
+<div class="user_show_container">
+<div class="user_bio">
+  <h3>Bio</h3>
+  <p><%= @user.bio %></p>
+</div>
+
+<% @user.most_recent_post.each_with_index do |post, index| %>
+  <div class="post_container">
+  <h4><%= post.class %> #<%=index%></h4>
+  <p><%= post.title %></p>
+  <div class="counters">
+    <span>Comments: <%= post.comments_counter %></span> 
+    <span>Likes: <%= post.likes_counter%> </span>
+  </div>
+  </div>
+<% end %>
+
+<button>See all posts</button>
```

Comment:
- [ ] Hi @Strangeal, let's kindly learn how to link our pages as requested in the requirements. Clicking on this button should display the user's post index page(users/:user_id/posts/).